sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -92,8 +92,24 @@ class ServiceBusMessage(
 
     """
 
-    def __init__(self, body, **kwargs):
-        # type: (Optional[Union[str, bytes]], Any) -> None
+    def __init__(
+        self,
+        body: Optional[Union[str, bytes]],
+        *,
+        application_properties: Optional[dict] = None,
```

Comment:
This should be `typing.Dict[?, ?]` (Not sure if there's a particular structure to application properties.... maybe the keys can be bytes? str? So maybe something like Dict[str, Any]?)